src/patterns/page-not-found-pages/index.md.njk

```diff
@@ -3,7 +3,7 @@ title: Page not found pages
 description: A page not found tells someone we cannot find the page they were trying to view. They are also known as 404 pages.
 section: Patterns
 theme: Pages
-aliases:
+aliases: '404'
```

Comment:
looks like our other aliases don't have quote marks, are they needed here?

Reply:
Yep. It didn't work without.